cairo_prover/Cargo.toml

```diff
@@ -6,9 +6,9 @@ version.workspace = true
 edition.workspace = true
 license.workspace = true
 
-# [[bin]]
-# name = "cairo-platinum"
-# path = "src/main.rs"
+[[bin]]
+name = "cairo-platinum"
+path = "src/main.rs"
```

Comment:
There was a recent pr merged to main that modified the Makefile to use `cairo-platinum-prover` as the name of the binary. That should be reverted if this change is merged to main